sources/platform/actors/running/input_and_output.md

```diff
@@ -50,8 +50,72 @@ You can quickly access the Actor's output from the run detail page:
 
 ![Actor output](./images/input_and_output/actor-output.png)
 
-And to access all the data associated with the run, see the **Storage** tab, where you can explore the Actor's default [Dataset](../../storage/dataset), [Key-value store](../../storage/key-value-store), and [Request queue](../../storage/request-queue):
-
-![Actor output](./images/input_and_output/actor-storage.png)
+And to access all the data associated with the run, see the **Storage** tab, where you can explore the Actor's default [Dataset](../../storage/dataset), [Key-value store](../../storage/key-value-store), and [Request queue](../../storage/request-queue).
 
 You can also use [API](https://docs.apify.com/api/v2) to retrieve the output. To learn more about this, read the [Run an Actor or task and retrieve data via API](/academy/api/run-actor-and-retrieve-data-via-api) tutorial.
+
+## Example
+
+Here is a simple Actor that demonstrates how to access input and produce output. The Actor takes two numbers as input, adds them together, and pushes the result to its default dataset.
+
+### Input
+
+To create a user interface for this input in the Apify Console, you would define the following `INPUT_SCHEMA.json`:
+
+```json title="INPUT_SCHEMA.json"
+{
+    "title": "Number adder",
```

Comment:
unsure what's the indentation convention for JSON, whether 2 or 4, or maybe we're not consistent about this across the docs